doc/new_user.rst

```diff
@@ -0,0 +1,110 @@
+Setting up a new user
```

Comment:
Maybe this should be "new user basics" or something? "Setting up a new user" sounds like an action an administrator would do